test-suite/Makefile

```diff
@@ -420,8 +421,32 @@ $(addsuffix .log,$(wildcard output/*.v)): %.v.log: %.v %.out $(PREREQUISITELOG)
 	  fi; \
 	} > "$@"
 
+$(addsuffix .log,$(wildcard output-coqtop/*.v)): %.v.log: %.v %.out $(PREREQUISITELOG)
+	@echo "TEST      $< $(call get_coq_prog_args_in_parens,"$<")"
+	$(HIDE){ \
+	  echo $(call log_intro,$<); \
+	  output=$*.out.real; \
+	  $(coqtop) < "$<" $(call get_coq_prog_args,"$<") 2>&1 \
+	    | grep -v "Welcome to Coq" \
```

Comment:
I think it is not a good idea to strip the output. Sure, this "header" will change, and the .out files will need to be updated, but I prefer the test to be pedantic rather than possibly incomplete (eg skipping warnings  seems just wrong).

I don't get exactly what the `File..` part does and why.

Reply:
I just copied this from the code for the `output` directory.  Exactly which lines would you like me to remove?  Certainly we want to remove the "Welcom to Coq" line because it has the version number, which will change.